### PR TITLE
Legacy URL redirects in Nginx

### LINKED ISSUE #306: Add legacy url redirects after runtime guide refactor
Add redirects for old URLs generated in boosters.

For example:

http://appdev.openshift.io/docs/mission-http-api-vertx.html

to 

http://appdev.openshift.io/docs/vertx-runtime.html#mission-http-api-vertx

### PR DESCRIPTION
**Pull request by tradej**
Can you please test on your machine with http://localhost:8080/docs/mission-http-api-spring-boot-tomcat.html ? Thank you. I want to verify this link works properly.

Resolves #306.

### CODE REVIEW

**Comment by gastaldi** [comment]
The URL you provided gives a 404 whilte attempting to redirect to the following URL: 

http://localhost:8080/docs/spring-boot-tomcat-runtime.html#mission-http-api-spring-boot-tomcat

**Comment by gastaldi** [comment]
However http://localhost:8080/docs/mission-http-api-vertx.html works as described in the original issue

File: nginx.conf [hunk 1]
```diff
@@ -59,6 +59,11 @@ http {
             return 301 /docs;
         }
 
+        # Redirects from the old URLs generated by the boosters
+        location ~* /docs/mission-([^/]+)-(wf-swarm|vertx|spring-boot-tomcat).html {
```
Comment (rhoads-zach) [inline nginx.conf]: I think spring-boot-tomcat should be spring-boot

Comment (rhoads-zach) [inline nginx.conf]: i think @gastaldi came across this as well.

**Comment by tradej** [comment]
I thought as much. Originally, I had a separate rule for Spring Boot, which worked, but then I removed it and it kept working (even after I rebuilt with `cico_build_deploy.sh` and restarted the container), so I was wondering what's up. A fix is imminent.

**Comment by tradej** [comment]
@rhoads-zach, please re-review. Thanks.

**Comment by rhoads-zach** [comment]
thank you @tradej. nice work! LGTM, merging

**Comment by rhoads-zach** [comment]
merged: 5f2add85d109b5489d338219f7d45237898e1c44

**Pull request closed without merging**